uiowa/uiowa
# Toggle Navigation Alignment

# Problem
If a site is using the toggle menu without links in the header, the right alignment disappears below the container breakpoint. 

<img width="1260" alt="Screen Shot 2021-02-15 at 3 42 51 PM" src="https://user-images.githubusercontent.com/1036433/107996094-2d95a180-6fa5-11eb-9902-f2dcfb4a3437.png">

This PR tracks changes for https://github.com/uiowa/uiowa/issues/3080

# How to test

1. Pull down this PR
2. Sync locally `writinguniversity.org`
3. run `blt frontend` in your vagrant VM
4. test all combinations of Iowa Bar settings at `Appearance > Settings > UIDS Base`. None of them should create the effect observed in the `Problem` Section.
